### ISSUE TITLE
Shield block not displaying correctly in defense window

### ISSUE THREAD

**Issue by Tarius-TGC** [nsweeting2/XaosUI#13]
Example image below:

<blockquote class="imgur-embed-pub" lang="en" data-id="a/CYpdFEf"  ><a href="//imgur.com/a/CYpdFEf">shieldblock example</a></blockquote><script async src="//s.imgur.com/min/embed.js" charset="utf-8"></script>

**Comment by lautensack** [comment]
Noah, it looks like there's an extra <TR> tag in the markup that's causing the issue. 
Find the `local ShieldBlockRoll =` section in the UI.UpdateLabelDefense script and replace that code chunk with the following for a quick fix.



```
 local ShieldBlockRoll =
    {
      "<td>&nbsp;</td>",
			"<td align=\"right\"><span style=\"color: " ..
      C2 ..
      ";\">Shield Block<span style=\"color: " ..
      C1 ..
      ";\">:</td>",
      "<td>" .. SHIELDBLOCK .. "</td>",
    };
```

**Comment by lautensack** [comment]
Issue Fixed in 2.0.0.5
